src/ir/from_ast/build_ctx.rs

```diff
@@ -59,6 +68,27 @@ impl std::ops::Index<&Id> for SigMap {
     }
 }
 
+impl std::iter::FromIterator<(Id, Sig)> for SigMap {
```

Comment:
I think fromiterator is in the prelude and doesn’t need to be imported from std::iter?